site/docs/.vuepress/components/LanguagePopup.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts" setup>
+import { reactive, computed } from "vue";
+
+const languages = {
+  es: "Spanish",
+  id: "Indonesian",
+  zh: "Chinese",
+};
+
+const popup = reactive({
+  enabled: localStorage.getItem("disable_language_bar") == null,
+});
+
+const language = computed(() =>
+  popup.enabled && navigator.language in languages
+    ? [navigator.language, languages[navigator.language]]
+    : null
```

Comment:
I think we need to check all `navigator.languages` here

Reply:
What if it included all the supported languages?

Reply:
Take the first.